sql/core/src/test/resources/sql-tests/results/postgreSQL/groupingsets.sql.out

```diff
@@ -444,10 +444,29 @@ NULL	1
 
 
 -- !query 34
-select count(*) from gstest4 group by rollup(unhashable_col,unsortable_col)
+select ten, sum(distinct four) filter (where string(four) like '123') from onek a
+group by rollup(ten)
 -- !query 34 schema
-struct<count(1):bigint>
+struct<ten:int,sum(DISTINCT four) FILTER (WHERE CAST(four AS STRING) LIKE 123):bigint>
 -- !query 34 output
```

Comment:
Checked, too: https://github.com/postgres/postgres/blob/0fca3d0a4ec297bff5a5cb01dfe345e0f63d7d63/src/test/regress/expected/groupingsets.out#L822

Reply:
The same